pkgs/development/haskell-modules/configuration-common.nix

```diff
@@ -1005,7 +1005,7 @@ self: super: {
   spdx = dontCheck super.spdx;
 
   # The test suite does not know how to find the 'alex' binary.
-  alex = overrideCabal super.alex (drv: {
+  alex = overrideCabal (appendPatch super.alex ./patches/alex-3.2.4-ghc8.8.x.patch) (drv: {
```

Comment:
Maybe this patch could just be applied in the `configuration-ghc-8.8.x.nix` file?  That would make it a little easier to remember to remove when an new release of `alex` is made.

Also, in general, people don't like carrying around patches in this `nixpkgs` repo if it is at all possible to just fetch them from upstream.  You can find a bunch of examples of how to do this in `configuration-ghc-8.8.x.nix` (all the places using `fetchpatch`).

Reply:
Good point, I will fix this!

Yeah, the locally stored patch feels a little dirty. I can put a comment there that the patch should be removed once version is bumped & released.